priam/src/main/java/com/netflix/priam/aws/AWSIPConverter.java

```diff
@@ -0,0 +1,58 @@
+package com.netflix.priam.aws;
+
+import com.google.common.base.Splitter;
+import com.netflix.priam.identity.PriamInstance;
+import com.netflix.priam.identity.config.InstanceInfo;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import javax.inject.Inject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AWSIPConverter implements IPConverter {
```

Comment:
nit: good to add comments to public classes

Reply:
Done.